sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_joined_v1/schema.yaml

```diff
@@ -320,6 +320,9 @@ fields:
   mode: REPEATED
   name: experiments
   type: RECORD
+- mode: NULLABLE
+  name: firefoxsuggest_optin
```

Comment:
I feel like this name should be a bit more verbose; just calling it `optin` sounds like a boolean to me. How about `firefoxsuggest_opt_in_dialog_choice`?